README.md

```diff
@@ -136,6 +136,10 @@ chmod a+x .git/hooks/pre-commit
 You can also setup the [IntelliJ plugin](https://plugins.jetbrains.com/plugin/8527)
 to reformat code from within your IDE
 
+### Lombok
+This project uses [Lombok](https://projectlombok.org/) to reduce boilerplate. You can setup
+ the [IntelliJ plugin](https://plugins.jetbrains.com/plugin/6317) to add IDE support. 
```

Comment:
I didn't know that, thanks for the perspective. Things do get gnarly in the IDE if one doesn't have the plugin installed (for IntelliJ), or the jar setup (for Eclipse); but once the one time setup is done, it's usually pretty smooth sailing. 
Do you recommend any alternatives?